layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,226 @@
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
+    </ul>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+  <p>In Node.js, we recognize that there is still an incredible amount of work to be accomplished to address the injustices and inequalities that exist, not only in society as a whole but within the technology industry, and even within our own project. To this point, we have not done enough and recognize that we need to do more. We commit to supporting the Black members of the community by using our collective platforms to amplify your words, or by helping you in your careers, contributing to open source, and seeking other opportunities to help.</p>
+
+ <p>Beginning with our upcoming Collaborator Summit event, and extending beyond, we will be offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If you are interested in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will post more information on how we will be proceeding with this effort as we finalize the details both for the period of the Collaborator summit and beyond that single event.</p>
```

Comment:
```suggestion
<p> We are offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If interested, please send an email to <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or a direct message to <a href="https://twitter.com/nodejs">@nodejs on Twitter</a>.</p>
```

Reply:
I'm in favor of this, but we should definitely think about how we're going to handle it if demand overwhelms mentoring resources, as has happened with the existing mentorship program where the ratio appears to be hundreds of applicants per mentor. I like being ambitious but I'm also wary of making promises we can't keep.

Reply:
The thought I had with this was to have something along the lines of a calendar that we could set up a calendly for and have 1:1 sessions with Black folks during the Collab Summit. This allows for more flexibility instead of "you need to be here at this time" and does allow us to scope to the times that we will be able to have a mentor available. Can go ahead and do a Doodle for folks who will be available to mentor. From there, we can begin to do the same outside of that and iterate from there.

Reply:
I would like to keep the initial copy. I am personally dedicating my time to helping with this and will be setting up a calendly page next week.

Reply:
>I would like to keep the initial copy. I am personally dedicating my time to helping with this and will be setting up a calendly page next week.

FWIW if it wasn't clear: I am also going to be dedicating my time to this.

Reply:
Resolving this for now, we can revisit after launch